README.md

```diff
@@ -1 +0,0 @@
-# Past-Project-Review
```

Comment:
Can you please update your read-me file with details of your project because it is a form of documentation for your project. It provides essential information about the project's purpose, functionality, and usage for the users, so it is a really important part of the project.